TESTS/block_device/qspif/main.cpp

```diff
@@ -0,0 +1,368 @@
+#include "greentea-client/test_env.h"
```

Comment:
Add license header here on the top of the file

Reply:
done